### ISSUE TITLE
Body of last commit is not visible with VerticalReverse orientation

### ISSUE THREAD

**Issue by MartinJohns** [nicoespeon/gitgraph.js#374]
**Describe the bug**

The body of the last commit is cut off and not shown when using the `VerticalReverse` orientation.

**To Reproduce**

Code:

```javascript
const graphContainer = document.getElementById("graph-container");
 
// Instantiate the graph.
const gitgraph = GitgraphJS.createGitgraph(graphContainer, { orientation: GitgraphJS.Orientation.VerticalReverse });

// Simulate git commands with Gitgraph API.
const master = gitgraph.branch("master");
master.commit({ subject: "Initial commit", body: "This is shown." });

const develop = gitgraph.branch("develop");
develop.commit({ subject: "Add TypeScript", body: "This is shown as well." });

master.commit({ subject: "Test commit", body: "This isn't shown! :-(" });

// Same applies for a merge:
//master.merge({ branch: develop, commitOptions: { body: "This isn't shown! :-(" } });
```

**Expected behavior**

The last commit shows a body.

**Screenshots**

![Image showing the issue](https://i.imgur.com/4yi5TRr.png)

**Context**

- Impacted library: `@gitgraph/js`
- Version: Latest delivered via `cdn.jsdelivr.net`
- Browser: Chrome 84, Edge 84

**Additional information**

No further information.

**Comment by MartinJohns** [comment]
The body is also cut off on the right side if the author is hidden, or if a long tag name is applied.

**Comment by andreasonny83** [comment]
A fix is coming in #386 @MartinJohns
